packages/stark-ui/src/modules/table/components/column.component.ts

```diff
@@ -253,6 +253,10 @@ export class StarkTableColumnComponent extends AbstractStarkUiComponent implemen
 		) {
 			this._filterFormCtrl.setValue(this.filterValue);
 		}
+
+		if (simpleChanges["sortPriority"] && typeof simpleChanges["sortPriority"].currentValue === "undefined") {
```

Comment:
Just to keep it align with the top code, 
could we put the "if" statement like this: 
if (
    simpleChanges["sortPriority"] && 
    typeof simpleChanges["sortPriority"].currentValue === "undefined"
) {

Reply:
Actually I formatted the code with Prettier and because the line of the condition is less than 140 characters, Prettier kept it in one line. I think this should stay like this.

Reply:
It was just a bit awkward when looking at it :-). 
But Prettier is the king 👍 .